packages/@lwc/engine/src/framework/base-lightning-element.ts

```diff
@@ -487,6 +487,12 @@ BaseLightningElementConstructor.prototype = {
         return elm.getElementsByClassName(names);
     },
 
+    get isConnected(): boolean {
+        const vm = getAssociatedVM(this);
+        const { elm } = vm;
+        return elm.isConnected;
```

Comment:
`isConnected` is polyfiled in @lwc/synthetic-shadow https://github.com/salesforce/lwc/blob/master/packages/@lwc/synthetic-shadow/src/faux-shadow/node.ts#L470